src/devices/MemoryLCD/samples/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.IO;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Net.Sockets;
+using System.Net;
+using System.Threading;
+using System.Drawing;
+using Iot.Device.MemoryLcd;
+
+namespace MemoryLcd.Samples
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            // LSxxxB7DHxx's chip select is high-level votage enabled
+            // You can use default ChipSelectLineActiveState value and a NOT gate to inverse this signal
+            // If you use gpio pins to control SCS, ChipSelectLineActiveState is optional
+            SpiDevice spi = SpiDevice.Create(new SpiConnectionSettings(0, 0)
+            {
+                ChipSelectLineActiveState = 1, // optional
+                ClockFrequency = 2_000_000,
+                DataFlow = DataFlow.MsbFirst,
+                Mode = 0
+            });
+
+            // You can fix DISP and EXTCOMIN in your circuit and use SPI's CE line to economize the GPIO pins
+            GpioController gpio = new GpioController(PinNumberingScheme.Logical);
+            LSxxxB7DHxx mlcd = new LS027B7DH01(spi, gpio, 25, 24, 23);
+
+            Random random = new Random();
+
+            Console.WriteLine("Clear. Press any key to continue...");
+            Console.ReadKey();
+
+            mlcd.AllClear();
+
+            Console.WriteLine("DataUpdate1Line. Press any key to continue...");
+            Console.ReadKey();
+
+            byte[] lineBuffer = new byte[mlcd.BytesPerLine];
+            for (int i = 0; i < mlcd.PixelHeight; i++)
+            {
+                random.NextBytes(lineBuffer);
```

Comment:
Would it be possible to provide (as a separate class or maybe as extension methods) a high-level interface to this, which allows writing text or bitmap instances directly? This low-level interface that doesn't even define the pixel format to use is not really nice to use.

Reply:
yes, and low-level interfaces also need to be retained for more efficient control